build.sbt

```diff
@@ -2,7 +2,7 @@ name := "dfdl-mil-std-2045"
 
 organization := "com.owlcyberdefense"
 
-version := "1.2.0"
+version := "1.3.0"
```

Comment:
I usually prefer to bump the version as a separate commit. It's probably not a big deal in this case, but if a commit included a version bump is ever reverted using git revert, then you lose the version bump which is probably not intended.